README.md

```diff
@@ -1,3 +1,2 @@
 # Northcoders News API
-
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+To access the database, you will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=, with the correct database name for that environment (see /db/setup.sql for the database names).
```

Comment:
You should explicitly state the database names in this readMe, Ie add PGDATABASE=nc_news; 
and PGDATABASE=nc_news_test; 

It's really easy for the user to forget the semicolons here, or they might simply make a mistake. Make this change and merge your work, without submitting for another review 👍